app/react/V2/Routes/Settings/Settings.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useMemo } from 'react';
```

Comment:
I think this route too warrants a test. Again, maybe we can have a task issue for it.